barretenberg/cpp/scripts/ci_benchmark_ivc_flows.sh

```diff
@@ -44,86 +46,72 @@ function verify_ivc_flow {
   fi
 }
 
-function client_ivc_flow_native {
-  set -eu
-  local flow=$1
-  local flow_folder="$input_folder/$flow"
-  local start=$(date +%s%N)
-  mkdir -p "bench-out/$flow-proof-files"
-  export MEMUSAGE_OUT="bench-out/$flow-proof-files/peak-memory-native-mb.txt"
-
-  function bb_cli_bench_native {
-    export MAIN_ARGS="$*"
-    memusage ./build/bin/bb_cli_bench \
-        --benchmark_out=bench-out/$flow-proof-files/op-counts.json \
-        --benchmark_out_format=json || {
-      echo "bb_cli_bench failed with args: $*"
+function run_bb_cli_bench {
+  local runtime="$1"
+  local output="$2"
+  local args="$3"
+  export MAIN_ARGS="$args"
+
+  if [[ "$runtime" == "native" ]]; then
+    memusage "./$native_build_dir/bin/bb_cli_bench" \
+      --benchmark_out=$output/op-counts.json \
+      --benchmark_out_format=json || {
+      echo "bb_cli_bench native failed with args: $args"
       exit 1
     }
-  }
-
-  bb_cli_bench_native prove -o "bench-out/$flow-proof-files" -b "$flow_folder/acir.msgpack" -w "$flow_folder/witnesses.msgpack" --scheme client_ivc --input_type runtime_stack
-  local end=$(date +%s%N)
-  local elapsed_ns=$(( end - start ))
-  local elapsed_ms=$(( elapsed_ns / 1000000 ))
-  local memory_taken_mb=$(cat "$MEMUSAGE_OUT")
```

Comment:
As noted mostly a refactor, but allows honouring NATIVE_PRESET=op-count-time now with op count times printing. We may just make op counts part of core bb as they are quite light-weight